api/nodes/views.py

```diff
@@ -22,10 +23,14 @@ class NodeMixin(object):
     node_lookup_url_kwarg = 'node_id'
 
     def get_node(self):
-        obj = get_object_or_error(Node, self.kwargs[self.node_lookup_url_kwarg], 'node')
+        node = get_object_or_error(Node, self.kwargs[self.node_lookup_url_kwarg], 'node')
```

Comment:
Try to avoid treating kwargs as positional where possible. The third argument should be `display_name='node'`